### ISSUE TITLE
Explicit documentation of which version of concurrent.futures is backported

### ISSUE THREAD

**Issue by snakescott** [agronholm/pythonfutures#73]
Given the version number I assume this is a backport of concurrent.futures from Python 3.2. There have been improvements to the module in later releases of Python 3, and I was wondering if there are any plans to pull them in? I'm specifically interested in improvements to [ProcessPoolExecutor](https://docs.python.org/3.7/library/concurrent.futures.html#concurrent.futures.ProcessPoolExecutor), namely 

1. the improved behavior on abrupt worker termination
2. initializer / initargs support

Thanks!

**Comment by agronholm** [comment]
Yes, the README does clearly state that this is a backport. As for the other issues:

1. Not doable (requires changes to Python core)
2. I can probably add this

**Comment by agronholm** [comment]
Also note that `ProcessPoolExecutor` is kinda broken due to problems that have been fixed way back in Python 3.3 but not 2.7.
